backend/src/test/java/ai/verta/modeldb/LineageTest.java

```diff
@@ -360,6 +381,9 @@ public void createAndDeleteLineageTest() throws ModelDBException, NoSuchAlgorith
 
     long repositoryId =
         RepositoryTest.createRepository(versioningServiceBlockingStub, RepositoryTest.NAME);
+    long repositoryId2 =
+        RepositoryTest.createRepository(
+            versioningServiceBlockingStubClient2, RepositoryTest.NAME + "2");
```

Comment:
I think the below test cases do not work well with OSS. please check it with OSS once.

Reply:
It works with OSS. There are some cases that are disabled in OSS test run by
`          if (app.getAuthServerHost() != null && app.getAuthServerPort() != null) {
`
but besides that everything will work